README.md

```diff
@@ -23,10 +23,14 @@ This is a cli tool to extract table data from a fits file and write it down in a
 | Postgres |  under validation |
 
 ## Installation 
-For installation you can use pip
+For installation up to version 0.0.3 can be installed with pip
 ```bash 
 pip install fits2db
 ```
+For newer verions use the git repository
+```bash
+pip install git+https://github.com/pmodwrc/fits2db.git@main
+```
 check if you got the right version with 
```

Comment:
Why not update the version on pypi? If you're the new maintainer on pmod I can add you to the mantainers of the package on pypi if you give me your pypi username